### DISCUSSION TITLE
input otp package

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#8830] [General]
Interesting new package made by @ritikbanger, if you are building a OTP form, worth to give it a short and ofc star the package if you find it useful. <3

https://github.com/ritikbanger/react18-input-otp

**Comment by ritikbanger** [comment]
Could you please edit the opt form to otp form.

**Reply by Moshyfawn** [reply]
Done ;)

**Reply by bluebill1049** [reply]
oops... thanks <3
